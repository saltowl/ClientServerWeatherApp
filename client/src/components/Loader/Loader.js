import React from 'react';
import './Loader.scss';

class Loader extends React.Component {
  constructor(props) {
    super(props);
  }
  
  render() {
    return (
      <div className="Loader">
        <div>Please wait, data is loading</div>
        <div><i className="fas fa-spinner fa-pulse"></i></div>
      </div>
    );
  }
}

export default Loader;
