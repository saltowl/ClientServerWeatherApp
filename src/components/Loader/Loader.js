import React from 'react';

class Loader extends React.Component {
  constructor(props) {
    super(props);
  }
  
  render() {
    return (
      <div className="Loader"/>
    );
  }
}

export default Loader;
