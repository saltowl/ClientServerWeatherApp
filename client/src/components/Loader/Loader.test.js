import React from "react";
import Loader from "./Loader";
import renderer from "react-test-renderer";

it("Loader display correctly", () => {
  const message = 'data is loading';
  const component = renderer.create(<Loader message={message} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
