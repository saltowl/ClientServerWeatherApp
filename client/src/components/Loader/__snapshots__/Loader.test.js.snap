// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader display correctly 1`] = `
<div
  className="Loader"
>
  <div>
    Please wait, 
    data is loading
     
  </div>
  <div>
    <i
      className="fas fa-spinner fa-pulse"
    />
  </div>
</div>
`;
